test/support/test_helpers.ex

```diff
@@ -1,15 +1,33 @@
 defmodule RemoteRetro.TestHelpers do
   use Wallaby.DSL
-  alias RemoteRetro.Repo
+  alias RemoteRetro.{Repo, User}
 
   def persist_idea_for_retro(context) do
-    %{idea: idea, retro: retro} = context
-    idea = Map.put(idea, :retro_id, retro.id)
+    %{idea: idea, retro: retro, user: user} = context
+    user =
+      if Map.has_key?(user, :id) do
```

Comment:
As a follow-up, if we persist a user specifically for a test, we should have access to that user via the test;s context (per the `persist_user_for_retro` helper below). So I think we'll always have access to an id. If that turns out to be true--I'm often wrong--let's get rid of the conditional here.